### PR TITLE
js.instanceof

### LINKED ISSUE #4: Add js.instanceof function
Should have `js.instanceof` function

### PR DESCRIPTION
**Pull request by giann**
closes #4

### CODE REVIEW

**Comment by daurnimator** [comment]
Only thing is if you could change the commit message to the existing style?

**Comment by giann** [comment]
renamed

File: src/jslib.js [hunk 1]
```diff
@@ -368,6 +368,13 @@ let jslib = {
 		let u = checkjs(L, 1);
 		lua.lua_pushnumber(L, +u);
 		return 1;
+	},
+	"instanceof": function(L) {
+		let u1 = checkjs(L, 1);
```
Comment (daurnimator) [inline src/jslib.js]: This needs to accept anything. not just JS objects

Comment (daurnimator) [inline src/jslib.js]: (use `tojs` for the first operand)

Comment (daurnimator) [inline src/jslib.js]: Hrm. maybe `tojs` isn't perfect either: I think we don't want the fallthrough/default case.

Comment (daurnimator) [inline src/jslib.js]: Perhaps:
```js
let u1 = testjs(L, idx);
if (u1 === void 0) {
   // push false
}
```

**Comment by daurnimator** [comment]
Sigh... sorry, looks like we want `tojs` after all.

```
> class foo {  
...   static [Symbol.hasInstance](instance) {
.....     return true;
..... }
... }
[Function: foo]
> "abc" instanceof foo
true
> void 0 instanceof foo
true
``

**Comment by daurnimator** [comment]
And infact. we may want `tojs` for the second operand as well.

**Pull request merged**